src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -238,6 +242,13 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           onChange={togglePersonalisedSchedule}
           label="Bookmarked events"
         />
+
+        <ButtonNG
+          className="NavBarWrapper__close-button"
+          iconName={faTimes}
+          onClick={() => closeEventSchedule()}
+        />
+
```

Comment:
Might be good oppo to replace the other `Button` -> `ButtonNG` if that brings increased consistency and doesn't require too much development time.

Reply:
fixed in 5afa7cde5b6bd9d643aeb30b9b72287e5fc9bb5b